repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -0,0 +1,132 @@
+#########################
+# Gatling Configuration #
```

Comment:
A quick test showed that leaving out that defaults does not break anything (which was expected). It also seems that we do not need the empty sections.